roles/ovirt-image-template/README.md

```diff
@@ -28,6 +31,18 @@ Role Variables
 | template_disk_interface | virtio           | Interface of the template disk. |
 | template_timeout   | 600                   | Amount of time to wait for the template to be created. |
 | template_nics      | {name: nic1, profile_name: ovirtmgmt, interface: virtio} | List of dictionaries that specify the NICs of template. |
+| external_templates   | UNDEF               | List of dictionaries that describes the templates |
```

Comment:
You can execute role multiple times to achieve that. But if you insist, then this isn't a material for this role.